index.html

```diff
@@ -136,57 +153,62 @@ <h4 class="card__canopy-title">CANOPY</h4>
-        </p>
-        <h2 class="about_contact__title">LET'S CONNECT</h2>
-        <ul class="about_portfolio__icons">
-            <li>
-                <a href="https://twitter.com/ahmedeid2684" rel="noopener noreferrer" target="_blank">
-                    <img src="images/twitter.svg" alt="twitter-logo">
-                </a>
-            </li>
-            <li>
-                <a href="https://www.linkedin.com/in/ahmed-eid-0018571b1/" rel="noopener noreferrer" target="_blank">
-                    <img src="images/linkedin.svg" alt="linkedin-logo">
-                </a>
-            </li>
-            <li>
-                <a href="https://github.com/ahmedeid6842" rel="noopener noreferrer" target="_blank">
-                    <img src="images/github.svg" alt="github-logo">
-                </a>
-            </li>
-        </ul>
-        <div class="about__btn">
-            <a href="https://drive.google.com/file/d/1b3KKgW82D9qDyDKbZ8u8TfOwGGFjV8Y1/view?usp=sharing"
-                class="btn ">get me resume</a>
+        <div class="about_basics">
+            <h2 class="about__title">About Myself</h2>
+            <p class="about_description">
+                Hello I’m a software developer! I can help you build a product , feature or website Look through some of
+                my work and experience! If you like what you see and have a project you need coded, don’t hesitate to
+                contact me.
+            </p>
+            <h2 class="about_contact__title">LET'S CONNECT</h2>
+            <ul class="about_portfolio__icons">
+                <li>
+                    <a href="https://twitter.com/ahmedeid2684" rel="noopener noreferrer" target="_blank">
+                        <img src="images/twitter.svg" alt="twitter-logo">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/ahmed-eid-0018571b1/" rel="noopener noreferrer"
+                        target="_blank">
+                        <img src="images/linkedin.svg" alt="linkedin-logo">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://github.com/ahmedeid6842" rel="noopener noreferrer" target="_blank">
+                        <img src="images/github.svg" alt="github-logo">
+                    </a>
+                </li>
+            </ul>
+            <div class="about__btn">
+                <a href="https://drive.google.com/file/d/1b3KKgW82D9qDyDKbZ8u8TfOwGGFjV8Y1/view?usp=sharing"
+                    class="btn ">Get me resume</a>
+            </div>
         </div>
         <ul class="drop__down">
             <li class="language__list">
                 <span>Languages</span>
                 <i class='fas fa-angle-down'></i>
             </li>
-            <li class="language_list_technologies">
-                <img src="./images/JavaScript.svg" alt="">
```

Comment:
- [ ] your mobile design for the technologies section is not consistent with the provided figma template please make sure to fix this as well :+1: 
- figma design 
![in-mf](https://user-images.githubusercontent.com/89055502/236356603-5e5acf37-33a5-4206-a208-a52985e8920b.png)
- your design
![in-m](https://user-images.githubusercontent.com/89055502/236356611-f5280062-71c1-472a-b827-e84c9082c546.png)